eng/pipelines/merge-docs-main-to-live.yml

```diff
@@ -0,0 +1,95 @@
+      AllowlistPath: docsms-allowlist/python-allowlist.txt
+    - RepoName: azure-docs-sdk-node
+      RepoOwner: MicrosoftDocs
+      SafeName: JS
+      AllowlistPath: docsms-allowlist/js-allowlist.txt
+
+- name: TargetBranch
+  type: string
+  default: live
+
+jobs:
+- ${{ each repo in parameters.DocsRepos }}:
+  - job: 'Release${{repo.SafeName}}DocsMs'
+    pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
+      vmImage: MMSUbuntu20.04
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+    steps:
+      # Checkout tool repo
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+      # Checkout docs repo
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          SkipDefaultCheckout: true
+          Repositories:
+          - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+            WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+          Paths:
+          - /*
+
+      - task: Powershell@2
+        inputs:
+          pwsh: true
+          filePath: eng/scripts/create-tag-for-current-branch.ps1
+          arguments: 
+            -GithubUrl "https://$(azuresdk-github-pat)@github.com/${{repo.RepoOwner}}/${{repo.RepoName}}.git" `
+            -WhatIf:$${{ parameters.WhatIfPreference }}
+          workingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+        displayName: Create tag for base branch
+
+      - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+        parameters:
+          WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+          DefaultBranchVariableName: ${{repo.RepoName}}
+
+      - task: Powershell@2
+        inputs:
+          pwsh: true
+          filePath: eng/scripts/validate-changes-on-allowed-path.ps1
+          arguments: 
+            -TargetBranch ${{ parameters.TargetBranch }} `
+            -BaseBranch '$(${{repo.RepoName}})' `
+            -AllowListPath "${{repo.AllowlistPath}}"
+          workingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+        displayName: Validate the changes within allowlist
+        condition: ne(variables['Skip.Validation'], 'true')
+
+      - task: Powershell@2
+        inputs:
+          pwsh: true
+          filePath: eng/scripts/sync-base-branch-to-target.ps1
+          arguments: 
+            -TargetBranch ${{ parameters.TargetBranch }} `
+            -BaseBranch "$(${{repo.RepoName}})" 
+          workingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+        displayName: Sync base branch to target
+
+      - template: /eng/common/pipelines/templates/steps/git-push-changes.yml
+        parameters: 
+          BaseRepoBranch: ${{ parameters.TargetBranch }}
+          BaseRepoOwner: ${{ repo.RepoOwner }}
+          TargetRepoOwner: ${{ repo.RepoOwner }}
+          TargetRepoName: ${{ repo.RepoName }}
+          SkipCheckingForChanges: true
+          CommitMsg: "Merge main to live $(Get-Date -Format 'yyyy-MM-dd')"
```

Comment:
Since we aren't committing anything I assume you don't need to set this parameter any longer.

Reply:
Ok, double check the yml, it is ok to use the default. Will remove this one.